packages/universal-components/src/SegmentedButton/SegmentedButton.js

```diff
@@ -18,11 +18,11 @@ export type Props = {|
   /**
    * Selected segment value
    */
-  +selectedValue: string,
+  +selectedValue: ?string,
   /**
    * Dispatched on segment press
    */
-  +onValueChange: (value: string) => void,
+  +onValueChange: (value: ?string) => void,
```

Comment:
You don't need `value` here, you can use just this: 
```
+onValueChange: ?string => void,
```

Reply:
Right, updated.